### DISCUSSION TITLE
TypeError: isContentEditable

### DISCUSSION THREAD

**Discussion by ManfredWisniewski** [uNmAnNeR/imaskjs#587] [Q&A] [answered]
I am using the ImaskMixin from the React Plugin and I can't find out how to solve the following error:

`TypeError: Cannot read properties of undefined (reading 'isContentEditable')`

I have set it up to extend a TextField like this:

```
import { IMaskMixin } from 'react-imask';

const MaskedTextField = IMaskMixin(({ ...props }) => <input {...props} />);

return (
    <MaskedTextField
           several_textField = props
          mask={'0000 0000 0000'}
          radix=" "
        />
)
```
Any ideas?

**Comment by uNmAnNeR** [comment, marked as answer]
how about

`const MaskedTextField = IMaskMixin(({ inputRef, ...props }) => <input {...props, ref: inputRef } />);`
